### PR TITLE
[Console] Add bright colors to console.

### PR DESCRIPTION
**Pull request by CupOfTea696**
Doc updates for symfony/symfony#39976

### CODE REVIEW

File: console/coloring.rst [hunk 1]
```diff
@@ -46,7 +46,9 @@ It is possible to define your own styles using the
     $output->writeln('<fire>foo</>');
 
 Any hex color is supported for foreground and background colors. Besides that, these named colors are supported:
-``black``, ``red``, ``green``, ``yellow``, ``blue``, ``magenta``, ``cyan`` and ``white``.
+``black``, ``red``, ``green``, ``yellow``, ``blue``, ``magenta``, ``cyan``, ``white``,
+``gray``, ``bright-red``, ``bright-green``, ``bright-yellow``, ``bright-blue``,
+``bright-magenta``, ``bright-cyan`` and ``bright-white``.
 
 .. versionadded:: 5.2
```
Comment (OskarStark) [inline console/coloring.rst:51]: Please add a "versionadded" directive like this below this line:

```rst
.. versionadded:: 5.3

    Support for bright colors was introduced in Symfony 5.3.
```

Thanks

**Review by OskarStark — CHANGES_REQUESTED** [review]

File: console/coloring.rst (hunk 1 above)
Comment (javiereguiluz) [reply console/coloring.rst:51]: Added while merging.

**Comment by javiereguiluz** [comment]
Frankie, thanks a lot for contributing this feature and its docs ... and congrats on your first Symfony Docs contribution 🎉

**Comment by CupOfTea696** [comment]
@javiereguiluz you're welcome, and thanks! :)

**Pull request merged**